R/dependencies.R

```diff
@@ -0,0 +1,410 @@
+##' @title Get the dependencies for a given report from the database
+##'
+##' @param name the name of the report
+##' @param id the id of the report, if omitted, use the id of the latest report
+##' @param upstream A boolean indicating if we want to move up or down the tree
```

Comment:
is there a case where we might want to do *both* directions? In that case, even if not implemented now we'd be better off with a character argument here (`direction = "upstream"` or `direction = "downstream"`)